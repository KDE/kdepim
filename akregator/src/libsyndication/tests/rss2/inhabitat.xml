<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet href="http://feeds.feedburner.com/~d/styles/rss2full.xsl" type="text/xsl" media="screen"?>
<?xml-stylesheet href="http://feeds.feedburner.com/~d/styles/itemcontent.css" type="text/css" media="screen"?>
<rss xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:sy="http://purl.org/rss/1.0/modules/syndication/" xmlns:admin="http://webns.net/mvcb/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:feedburner="http://rssnamespace.org/feedburner/ext/1.0" version="2.0">
  <channel>
    <title>Inhabitat</title>
    <link>http://www.inhabitat.com</link>
    <description>the future of design</description>
    <dc:language>eng</dc:language>
    <dc:creator />
    <dc:rights>Copyright 2006</dc:rights>
    <dc:date>2006-01-15T21:36:47-00:00</dc:date>
    <admin:generatorAgent rdf:resource="http://www.pivotlog.net/?ver=Pivot+-+1.24.1%3A+%27Arcee%27" />
    <admin:errorReportsTo rdf:resource="mailto:rsserrors@pivotlog.net" />
    <sy:updatePeriod>hourly</sy:updatePeriod>
    <sy:updateFrequency>1</sy:updateFrequency>
    <sy:updateBase>2000-01-01T12:00+00:00</sy:updateBase>
    <image>
      <link>http://www.inhabitat.com</link>
      <url>http://www.inhabitat.com/images/smalllogo.jpg</url>
      <title>INHABITAT</title>
    </image>
    <atom10:link xmlns:atom10="http://www.w3.org/2005/Atom" rel="self" href="http://feeds.feedburner.com/Inhabitat" type="application/rss+xml" />
    <feedburner:browserFriendly>This is an XML content feed. It is intended to be viewed in a newsreader or syndicated to another site, subject to copyright and fair use.</feedburner:browserFriendly>
    <item>
      <title>ELASTIC CO.</title>
      <link>http://www.inhabitat.com/entry_1030.php</link>
      <comments>http://www.inhabitat.com/entry_1030.php#comm</comments>
      <description>Of all of the materials I wouldn't want to wear, rubber is right up at the top. It doesn't breathe, it catches the little hairs on your arms...but creating dancers' costumes out of rubber bands led designer Elodie Blanchard to great inspirations for home decor, which she produces under the name ElasticCo. 

The most commercially appealing among her varied designs is the Mesh Collection, a line of tabletop accessories and lighting made with many small, colorful rubber bands.  The containers are great for holding perishables that need a little ventilation in order to stay fresh, and the lamps diffuse light beautifully onto surrounding surfaces. 

I wouldn't want to dance in a rubberband get-up, but I'd stash my fruit in an ElasticCo. bowl any day. 

+ elasticco.com</description>
      <guid isPermaLink="false">1030@http://www.inhabitat.com/</guid>
      <content:encoded><![CDATA[<p style="text-align:center;"><img src="http://www.inhabitat.com/images/bowls.jpg" border="0" title="" alt="" class="pivot-image" /></p><br />
Of all of the materials I wouldn't want to wear, rubber is right up at the top. It doesn't breathe, it catches the little hairs on your arms...but creating dancers' costumes out of rubber bands led designer Elodie Blanchard to great inspirations for home decor, which she produces under the name <a target="new" href="http://elasticco.com/">ElasticCo.</a> <br />
<br />
The most commercially appealing among her varied designs is the Mesh Collection, a line of tabletop accessories and lighting made with many small, colorful rubber bands.  The containers are great for holding perishables that need a little ventilation in order to stay fresh, and the lamps diffuse light beautifully onto surrounding surfaces. <br />
<br />
I wouldn't want to dance in a rubberband get-up, but I'd stash my fruit in an <a target="new" href="http://elasticco.com/">ElasticCo.</a> bowl any day. <br />
<br />
<a target="new" href="http://elasticco.com/">+ elasticco.com</a><br />
<br />
<i></i><br  /><p style="text-align:center;"><img src="http://www.inhabitat.com/images/lamps.jpg" border="0" title="" alt="" class="pivot-image" /></p><br />
<p style="text-align:center;"><img src="http://www.inhabitat.com/images/lemons.jpg" border="0" title="" alt="" class="pivot-image" /></p><br />
<p style="text-align:center;"><img src="http://www.inhabitat.com/images/lampies.jpg" border="0" title="" alt="" class="pivot-image" /></p> 
<p><map name="google_ad_map_368m0r5.LxiWpzOOXoeDcF-si38_"><area shape="rect" href="http://imageads.googleadservices.com/pagead/imgclick/368m0r5.LxiWpzOOXoeDcF-si38_?pos=0" coords="1,2,367,28"/><area shape="rect" href="http://services.google.com/feedback/abg" coords="384,10,453,23"/></map><img usemap="#google_ad_map_368m0r5.LxiWpzOOXoeDcF-si38_" border="0" src="http://imageads.googleadservices.com/pagead/ads?format=468x30_aff_img&client=ca-pub-4434321579063536&output=png&cuid=368m0r5.LxiWpzOOXoeDcF-si38_&url=http%3A%2F%2Fwww.inhabitat.com%2Fentry_1030.php"/></p>]]></content:encoded>
      <dc:subject>Home decor</dc:subject>
      <dc:date>2006-01-15T20:20:00-00:00</dc:date>
    </item>
  </channel>
</rss>

