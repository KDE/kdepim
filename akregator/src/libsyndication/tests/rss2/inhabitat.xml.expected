# Feed begin ######################
title: #Inhabitat#
link: #http://www.inhabitat.com#
description: #the future of design#
copyright: #Copyright 2006#
language: #eng#
# Image begin #####################
url: #http://www.inhabitat.com/images/smalllogo.jpg#
title: #INHABITAT#
link: #http://www.inhabitat.com#
height: #31#
width: #88#
# Image end #######################
# Item begin ######################
id: #1030@http://www.inhabitat.com/#
title: #ELASTIC CO.#
link: #http://www.inhabitat.com/entry_1030.php#
description: #Of all of the materials I wouldn't want to wear, rubber is right up at the top. It doesn't breathe, it catches the little hairs on your arms...but creating dancers' costumes out of rubber bands led designer Elodie Blanchard to great inspirations for home decor, which she produces under the name ElasticCo. <br/> <br/>The most commercially appealing among her varied designs is the Mesh Collection, a line of tabletop accessories and lighting made with many small, colorful rubber bands. The containers are great for holding perishables that need a little ventilation in order to stay fresh, and the lamps diffuse light beautifully onto surrounding surfaces. <br/> <br/>I wouldn't want to dance in a rubberband get-up, but I'd stash my fruit in an ElasticCo. bowl any day. <br/> <br/>+ elasticco.com#
content: #<p style="text-align:center;"><img src="http://www.inhabitat.com/images/bowls.jpg" border="0" title="" alt="" class="pivot-image" /></p><br /> Of all of the materials I wouldn't want to wear, rubber is right up at the top. It doesn't breathe, it catches the little hairs on your arms...but creating dancers' costumes out of rubber bands led designer Elodie Blanchard to great inspirations for home decor, which she produces under the name <a target="new" href="http://elasticco.com/">ElasticCo.</a> <br /> <br /> The most commercially appealing among her varied designs is the Mesh Collection, a line of tabletop accessories and lighting made with many small, colorful rubber bands. The containers are great for holding perishables that need a little ventilation in order to stay fresh, and the lamps diffuse light beautifully onto surrounding surfaces. <br /> <br /> I wouldn't want to dance in a rubberband get-up, but I'd stash my fruit in an <a target="new" href="http://elasticco.com/">ElasticCo.</a> bowl any day. <br /> <br /> <a target="new" href="http://elasticco.com/">+ elasticco.com</a><br /> <br /> <i></i><br /><p style="text-align:center;"><img src="http://www.inhabitat.com/images/lamps.jpg" border="0" title="" alt="" class="pivot-image" /></p><br /> <p style="text-align:center;"><img src="http://www.inhabitat.com/images/lemons.jpg" border="0" title="" alt="" class="pivot-image" /></p><br /> <p style="text-align:center;"><img src="http://www.inhabitat.com/images/lampies.jpg" border="0" title="" alt="" class="pivot-image" /></p> <p><map name="google_ad_map_368m0r5.LxiWpzOOXoeDcF-si38_"><area shape="rect" href="http://imageads.googleadservices.com/pagead/imgclick/368m0r5.LxiWpzOOXoeDcF-si38_?pos=0" coords="1,2,367,28"/><area shape="rect" href="http://services.google.com/feedback/abg" coords="384,10,453,23"/></map><img usemap="#google_ad_map_368m0r5.LxiWpzOOXoeDcF-si38_" border="0" src="http://imageads.googleadservices.com/pagead/ads?format=468x30_aff_img&client=ca-pub-4434321579063536&output=png&cuid=368m0r5.LxiWpzOOXoeDcF-si38_&url=http%3A%2F%2Fwww.inhabitat.com%2Fentry_1030.php"/></p>#
datePublished: #Sun Jan 15 21:20:00 2006#
dateUpdated: #Sun Jan 15 21:20:00 2006#
commentsLink: #http://www.inhabitat.com/entry_1030.php#comm#
# Item end ########################
# Feed end ########################

